addons/payment/tests/utils.py

```diff
@@ -40,10 +40,13 @@ def _extract_values_from_html_form(self, html_form):
         :rtype: dict[str:str]
         """
         html_tree = objectify.fromstring(html_form)
-        action = html_tree.get('action')
-        inputs = {form_input.get('name'): form_input.get('value') for form_input in html_tree.input}
+        if hasattr(html_tree, 'input'):
+            inputs = {input_.get('name'): input_.get('value') for input_ in html_tree.input}
+        else:
+            inputs = {}
```

Comment:
Same as above, should this not be a separated commit ?

Reply:
This change was made necessary by a test in `payment_flutterwave` so I think it's fine like this, in the same commit.